SwiftUI_Homework/SwiftUI_Homework/View/UpdateSetting.swift

```diff
@@ -0,0 +1,168 @@
+// UpdateSetting.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран "Настройки"
+struct UpdateSetting: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let updateSettingText = "Update Settings"
+        static let productSettingText = "Product Update"
+        static let commentsText = "Comments"
+        static let settingsText = "Settings apply"
+        static let offerText = "Offer update"
+        static let notificationText = "Notifications"
+        static let stairText = "Stair lifts free the freedom of your home"
+        static let avertisingText = "Advertising relationships vs business"
+        static let mediaText = "A right media mix can make"
+        static let remarkableText = "Creating remarkable poster prints"
+        static let chevronImageName = "chevron.left"
+        static let widthButtonValue: CGFloat = 200
+        static let heightButtonValue: CGFloat = 70
+        static let cornerRadiusValue: CGFloat = 30
+        static let widthImageValue: CGFloat = 130
+        static let heightImageValue: CGFloat = 130
+        static let paddingTop: CGFloat = 40
+        static let orangeGradient = LinearGradient(
+            colors: [.yellow, .red],
+            startPoint: .topLeading,
+            endPoint: .bottomTrailing
+        )
+    }
+
+    // MARK: - Public properties
+
+    var body: some View {
+        NavigationView {
+            ZStack {
+                Constants.orangeGradient.ignoresSafeArea(edges: .top)
+                Color.white
+                VStack {
+                    listView
+                    updateButtonView
+                }
+            }
+        }
+        .navigationBarTitle(Constants.notificationText)
+        .navigationBarItems(leading: chevronButtonView)
+        .navigationBarBackButtonHidden()
+    }
+
+    // MARK: - Private Properties
+
+    @Environment(\.presentationMode) private var presentation
+
+    @StateObject private var updateSettingModel = UpdateSettingModel()
+
+    private var chevronButtonView: some View {
+        Button(action: {
+            presentation.wrappedValue.dismiss()
+        }, label: {
+            Image(systemName: Constants.chevronImageName)
+                .foregroundColor(.black)
+        })
+    }
+
+    private var listView: some View {
```

Comment:
```suggestion
    private var listView: some View {
```
Надо разнести на мелкие вью